docker-compose.yml

```
@@ -1,9 +1,18 @@
+    command: pnpm exec next dev
     restart: unless-stopped
     volumes:
       - .:/app
     ports:
       - '3000:3000'
+    environment:
+      - NODE_ENV=development
+      - CHOKIDAR_USEPOLLING=true
+      - WATCHPACK_POLLING=true
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:3000" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)